doc/policy/mempool-limits.md

```diff
@@ -8,58 +8,12 @@ Tx0 is a *parent* of Tx1 and Tx1 is a *child* of Tx0.
 A transaction's *ancestors* include, recursively, its parents, the parents of its parents, etc.
 A transaction's *descendants* include, recursively, its children, the children of its children, etc.
 
-A mempool entry's *ancestor count* is the total number of in-mempool (unconfirmed) transactions in
-its ancestor set, including itself.
-A mempool entry's *descendant count* is the total number of in-mempool (unconfirmed) transactions in
-its descendant set, including itself.
+A transaction's *cluster* is the set of all transactions that can be reached
+from that transaction by following all parent or child relationships. For
```

Comment:
In commit "doc: Update mempool-limits.md for cluster mempool".

Perhaps worth elaborating that it is not just parents and children, but also any combination thereof:

> A transaction's *cluster* is the set of all transactions that can be reached by following any combination of parent or child relationships. This includes all ancestors and descendants, but also any descendants of those ancestors and ancestors of those descendants, and so on.